starwars_api/models.py

```diff
@@ -28,8 +33,11 @@ def all(cls):
         later in charge of performing requests to SWAPI for each of the
         pages while looping.
         """
-        pass
-
+        if cls.RESOURCE_NAME == 'people':
+            return PeopleQuerySet()
```

Comment:
wish this wasn't hardcoded (cos I have the same problem with my implementation and my code is identical to yours here).
but `cls.RESOURCE_NAME.capitalize()`returns `"People"` so that was a no-go.